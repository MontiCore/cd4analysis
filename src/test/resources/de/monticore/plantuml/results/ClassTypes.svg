<?xml version="1.0" encoding="us-ascii" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentStyleType="text/css" height="333px" preserveAspectRatio="none" style="width:429px;height:333px;background:#FFFFFF;" version="1.1" viewBox="0 0 429 333" width="429px" zoomAndPan="magnify"><defs/><g><rect fill="none" height="27.6094" id="_legend" style="stroke:#000000;stroke-width:1.0;" width="112" x="291" y="12"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="102" x="296" y="31.5332">.ClassTypes CD</text><!--class Abstrakt--><g id="elem_Abstrakt"><rect codeLine="19" fill="#FFFFFF" height="42.7031" id="Abstrakt" style="stroke:#181818;stroke-width:0.5;" width="60" x="7" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="58" x="8" y="77.0664">&#171;abstract&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="52" x="11" y="94.2363">Abstrakt</text></g><!--class Finale--><g id="elem_Finale"><rect codeLine="21" fill="#FFFFFF" height="42.7031" id="Finale" style="stroke:#181818;stroke-width:0.5;" width="45" x="102.5" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="37" x="106.5" y="77.0664">&#171;final&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="39" x="105.5" y="94.2363">Finale</text></g><!--class Statische--><g id="elem_Statische"><rect codeLine="23" fill="#FFFFFF" height="42.7031" id="Statische" style="stroke:#181818;stroke-width:0.5;" width="64" x="183" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="42" x="194" y="77.0664">&#171;static&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="58" x="186" y="94.2363">Statische</text></g><!--class Lokal--><g id="elem_Lokal"><rect codeLine="25" fill="#FFFFFF" height="42.7031" id="Lokal" style="stroke:#181818;stroke-width:0.5;" width="42" x="282" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="40" x="283" y="77.0664">&#171;local&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="34" x="286" y="94.2363">Lokal</text></g><!--class Public--><g id="elem_Public"><rect codeLine="27" fill="#FFFFFF" height="27.6094" id="Public" style="stroke:#181818;stroke-width:0.5;" width="44" x="15" y="170.1094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="38" x="18" y="189.6426">Public</text></g><!--class Protected--><g id="elem_Protected"><rect codeLine="29" fill="#FFFFFF" height="42.7031" id="Protected" style="stroke:#181818;stroke-width:0.5;" width="67" x="94.5" y="162.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="65" x="95.5" y="180.0664">&#171;protected&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="61" x="97.5" y="197.2363">Protected</text></g><!--class PAF--><g id="elem_PAF"><rect codeLine="31" fill="#FFFFFF" height="42.7031" id="PAF" style="stroke:#181818;stroke-width:0.5;" width="126" x="197" y="162.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="124" x="198" y="180.0664">&#171;private final abstract&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="27" x="246.5" y="197.2363">PAF</text></g><!--class RO--><g id="elem_RO"><rect codeLine="33" fill="#FFFFFF" height="42.7031" id="RO" style="stroke:#181818;stroke-width:0.5;" width="64" x="358" y="162.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="62" x="359" y="180.0664">&#171;readonly&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="21" x="379.5" y="197.2363">RO</text></g><!--class Derived--><g id="elem_Derived"><rect codeLine="35" fill="#FFFFFF" height="42.7031" id="Derived" style="stroke:#181818;stroke-width:0.5;" width="57" x="8.5" y="265.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="55" x="9.5" y="283.0664">&#171;derived&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="49" x="12.5" y="300.2363">Derived</text></g><!--class PDR--><g id="elem_PDR"><rect codeLine="37" fill="#FFFFFF" height="42.7031" id="PDR" style="stroke:#181818;stroke-width:0.5;" width="145" x="100.5" y="265.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="143" x="101.5" y="283.0664">&#171;public derived readonly&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="29" x="158.5" y="300.2363">PDR</text></g><!--class Interface--><g id="elem_Interface"><rect codeLine="39" fill="#FFFFFF" height="27.6094" id="Interface" style="stroke:#181818;stroke-width:0.5;" width="62" x="281" y="273.1094"/><text fill="#000000" font-family="sans-serif" font-size="14" font-style="italic" lengthAdjust="spacing" textLength="56" x="284" y="292.6426">Interface</text></g><text fill="#888888" font-family="sans-serif" font-size="10" lengthAdjust="spacing" textLength="200" x="107.5" y="323.6934">generated with MontiCore using PlantUML</text><!--SRC=[RP7XQeCm58L_cwUuJx3j0QJGLGg5bfLsOxzZlDMBCP7akk56tts9SSL2_Q7dvZldSjJSywXn9JAbfVS5oTh93dv5bWucbRrArTRE3gRQM6qTV3R46C4fdJc33bv41d2DDuqrcYf4OReUfnExNn1FhY9_dZirL24eSafZqjZrF4A7NOdEYs3tBPbUniPJspMpet9Wt2bhplI3OUTSXcqFZkg6G-ff4ytlOuyUDeM8EBb4UkTrwTd9bY7FGKvQCQnMOiPRCb9ZX9Ue5kJCaichPe8-3cf1zxODxa2qLNUnur1gKhV9MKR5M4LhVniMzlKs8VgIZ7E91prFRz7cK5RMw7412WpHjB-QvM9xSPgsJvrk_6uB6KPtaGfXzwz26gE6YxNn4RucEXbxVnCtSB26AVncXC6JgU6efU6Fm_uF]--></g></svg>
