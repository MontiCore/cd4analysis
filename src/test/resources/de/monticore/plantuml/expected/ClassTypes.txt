@startuml
  skinparam classAttributeIconSize 0
  skinparam legend {
    BorderColor black
    BackGroundColor white
  }
  legend top right
    .ClassTypes CD
  end legend
  class Abstrakt {
    {field} boolean abstrakt
  }

  class Finale {
    {field} String ohoho
  }

  class Statische {
    {field} int spannung
  }

  class Lokal

  class Privat

  class Public

  class Protected

  class PAF

  class RO

  class Derived

  class PDR

  interface Interface

  center footer generated with MontiCore using PlantUML
@enduml
